meta.go

```diff
@@ -0,0 +1,14 @@
+package ingresstpr
+
+// Metadata to create operatorkit TPR.
+
+const (
+	// Name represents the name of the third party resource within Kubernetes.
+	Name = "ingress-controller.giantswarm.io"
```

Comment:
I tend to think what I'd like to see when invoking `kubectl get`. In this case `kubectl get ingresscontrollers` - looks a bit awkward.

My proposition is `ingress-config`. Which is a kind `IngressConfig` in a `giantswarm.io` group. That makes sense for me.